lib/parse.js

```diff
@@ -2541,6 +2562,12 @@ function parse($TEXT, options) {
           case "atom":
             next();
             return tmp.value;
+          case "expand":
+            next();
+            return new AST_Expansion({
+                start: prev(),
+                expression: expression(false),
+            });
```

Comment:
There are multiple sites which calls `as_property_name`, e.g.
https://github.com/mishoo/UglifyJS2/pull/2265/files#diff-9f97a9c36fbbfb0d19068d6fae3bb84eL1567

Would you need to handle `AST_Expansion` on all of them?